modern/src/Actions.ts

```diff
@@ -66,12 +91,12 @@ export function gotSkinZip(zip: JSZip, store: ModernStore) {
             node,
             new Set(["group", "WinampAbstractionLayer", "WasabiXML"])
           );
```

Comment:
I wonder if we should (eventually) move this into the `System` constructor.

Reply:
hrmmm, probably! hah